wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -630,9 +659,16 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
 
         val blockchainIdentityData = blockchainIdentityDataDaoAsync.load() ?: return
         val userId = blockchainIdentityData!!.getIdentity(walletApplication.wallet) ?: return
+        if (blockchainIdentityData.username == null) {
+            return // this is here because the wallet is being reset without removing blockchainIdentityData
+        }
```

Comment:
There were bugs in the wallet that upon loading when the app started, the app would require a blockchain reset.  This is still a possibility, given the amount of changes that were required to support payments from and to contacts.